DigitalLearningSolutions.Data/DataServices/GroupsDataService.cs

```diff
@@ -116,5 +121,34 @@ FROM Groups
                 new { groupId, centreId }
             ).SingleOrDefault();
         }
+
+        public void RemoveRelateProgressRecordsForDelegate(int groupId, int delegateId, DateTime removedDate)

```

Comment:
Should "Relate" be replaced with "Group"?

Reply:
I had intended to call it RemoveRelatedProgressRecordsForDelegate. Now named RemoveRelatedProgressRecordsForGroupDelegate